src/flamenco/runtime/fd_borrowed_account.h

```diff
@@ -43,6 +46,29 @@ typedef struct fd_borrowed_account fd_borrowed_account_t;
 
 #define FD_BORROWED_ACCOUNT_DECL(_x)  fd_borrowed_account_t _x[1]; fd_borrowed_account_init(_x);
 
+/* This macro provides the same scoping guarantees as the Agave client's 
+   borrowed account semantics. It allows for implict/explicit dropping of 
+   borrowed accounts' write locks. It's usage mirrors the use of
+   FD_SCRATCH_SCOPE_{BEGIN,END}. It is also safe to use the original
+   acquire/release api within the scoped macro in case you don't want
+   variables to go out of scope. An example of this is in the extend
+   instruction within the bpf loader.  */
+#define FD_BORROWED_ACCOUNT_TRY_BORROW_IDX( _ctx, _idx, _account ) do {   \
+  fd_borrowed_account_t * _account = NULL;                                \
```

Comment:
This is wrong, it replaces the variable name at this line with the macro parameter.
Whatever name the user provides, it will be shadowed by this declaration

Reply:
Oh I guess that's the intended behavior? Resolving this then.